nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/shape/tensorops/EmbeddingLookup.java

```diff
@@ -0,0 +1,64 @@
+/* ******************************************************************************
+ * Copyright (c) 2020 Konduit K.K.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ******************************************************************************/
+package org.nd4j.linalg.api.ops.impl.shape.tensorops;
+
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import lombok.val;
+import org.nd4j.autodiff.samediff.SDVariable;
+import org.nd4j.autodiff.samediff.SameDiff;
+import org.nd4j.base.Preconditions;
+import org.nd4j.linalg.api.buffer.DataType;
+import org.nd4j.linalg.api.ndarray.INDArray;
+import org.nd4j.linalg.api.ops.DynamicCustomOp;
+import org.nd4j.linalg.factory.Nd4j;
+
+import java.util.Collections;
+import java.util.List;
+
+@NoArgsConstructor
+public class EmbeddingLookup extends DynamicCustomOp {
+
+    @Override
+    public String opName() {
+        return "embedding_lookup";
+    }
+
+
+    public EmbeddingLookup(@NonNull SameDiff sameDiff, @NonNull SDVariable in, @NonNull SDVariable indeces) {
+        super("embedding_lookup", sameDiff, new SDVariable[]{in, indeces});
+    }
+
+    public EmbeddingLookup(@NonNull INDArray in, @NonNull INDArray indeces, INDArray output) {
+        super("embedding_lookup", new INDArray[]{in, indeces}, new INDArray[]{output});
+    }
+
+    public EmbeddingLookup(@NonNull INDArray in, INDArray output, @NonNull int... indeces) {
+        super("embedding_lookup", new INDArray[]{in, Nd4j.create(indeces)}, new INDArray[]{output});
```

Comment:
Nd4j.create(int...) creates an Float/Double NDArray with _shape_ equal to the provided array, which is not what we want here. i.e., Nd4j.create(1,2,3,4,5) creates a rank 5 array with shape [1,2,3,4,5[

Instead: use Nd4j.createFromArray(int...) which creates a 1d integer array shape [array_length] with _values_ (1,2,3,4,5)